### PR TITLE
Update connect-to-remote-aadj-pc.md

### PR DESCRIPTION
**Pull request by RavennMSFT**
Updated supported configs to be more readable, added details on adding AAD users and groups to Remote Desktop Users group via policy, removed the Windows account picker text which is not supported for AAD users

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 1]
```diff
@@ -22,14 +22,15 @@ ms.topic: article
 
 - Windows 10
 
-From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](https://docs.microsoft.com/azure/active-directory/user-help/device-management-azuread-joined-devices-setup). Starting in Windows 10, version 1809, you can [use biometrics to authenticate to a remote desktop session](https://docs.microsoft.com/windows/whats-new/whats-new-windows-10-version-1809#remote-desktop-with-biometrics).
+From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](hhttps://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-join). Starting in Windows 10, version 1809, you can [use biometrics to authenticate to a remote desktop session](https://docs.microsoft.com/windows/whats-new/whats-new-windows-10-version-1809#remote-desktop-with-biometrics).
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:25]:
```suggestion
From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-join). Starting in Windows 10, version 1809, you can [use biometrics to authenticate to a remote desktop session](https://docs.microsoft.com/windows/whats-new/whats-new-windows-10-version-1809#remote-desktop-with-biometrics).
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 2]
```diff
@@ -22,14 +22,15 @@ ms.topic: article
...
 
 ![Remote Desktop Connection client](images/rdp.png)
 
 ## Set up
 
 - Both PCs (local and remote) must be running Windows 10, version 1607 or later. Remote connections to an Azure AD-joined PC running earlier versions of Windows 10 are not supported.
-- Your local PC (where you are connecting from) must be either Azure AD joined or Hybrid Azure AD joined if using Windows 10 version 1607 and above, or Azure AD registered if using Windows 10 version 2004 and above. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported. 
+- Your local PC (where you are connecting from) must be either Azure AD joined or Hybrid Azure AD joined if using Windows 10 version 1607 and above, or [Azure AD registered](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-register) if using Windows 10 version 2004 and above. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported.
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:32]:
```suggestion
- Your local PC (where you are connecting from) must be either Azure AD-joined or Hybrid Azure AD-joined if using Windows 10, version 1607 and above, or [Azure AD registered](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-register) if using Windows 10, version 2004 and above. Remote connections to an Azure AD-joined PC from an unjoined device or a non-Windows 10 device are not supported. 
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 3]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
-     > When you connect to the remote PC, enter your account name in this format: AzureAD\yourloginid@domain.com. The local PC must either be domain-joined or Azure AD-joined. The local PC and remote PC must be in the same Azure AD tenant.
+        This command only works for AADJ device users already added to any of the local groups (administrators).
+        Otherwise this command throws the below error. For example:
+        - for cloud only user: "There is no such global user or group : *name*"
+        - for synced user: "There is no such global user or group : *name*" </br>
 
-     > [!Note]
-     > If you cannot connect using Remote Desktop Connection 6.0, you must turn off the new features of RDP 6.0 and revert back to RDP 5.0 by making a few changes in the RDP file. See the details in the [support article](https://support.microsoft.com/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).
+         > [!NOTE]
+         > For devices running Windows 10, version 1703 or earlier, the user must to sign in to the remote device first before attempting remote connections.
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:61]:
```suggestion
         > For devices running Windows 10, version 1703 or earlier, the user must sign in to the remote device first before attempting remote connections.
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 4]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
+         >
+         > Starting Windows 10, version 1709, you can add other Azure AD users to the **Administrators** group on a device in **Settings** and restrict remote credentials to **Administrators**. If there is a problem connecting remotely, make sure that both devices are joined to Azure AD and that TPM is functioning properly on both devices.
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:63]:
```suggestion
         > Starting in Windows 10, version 1709, you can add other Azure AD users to the **Administrators** group on a device in **Settings** and restrict remote credentials to **Administrators**. If there is a problem connecting remotely, make sure that both devices are joined to Azure AD and that TPM is functioning properly on both devices.
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 5]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
 
-## Supported configurations
-
-In organizations using integrated Active Directory and Azure AD, you can connect from a Hybrid-joined PC to an Azure AD-joined PC by using any of the following:
+      - Adding users using policy
+     
+         Starting Windows 10 version 2004, you can add users or Azure AD groups to the Remote Desktop Users using MDM policies as described [in this article](https://docs.microsoft.com/azure/active-directory/devices/assign-local-admin#manage-administrator-privileges-using-azure-ad-groups-preview)
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:67]:
```suggestion
         Starting in Windows 10, version 2004, you can add users or Azure AD groups to the Remote Desktop Users using MDM policies as described in [How to manage the local administrators group on Azure AD joined devices](https://docs.microsoft.com/azure/active-directory/devices/assign-local-admin#manage-administrator-privileges-using-azure-ad-groups-preview).
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 6]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
 
-- Password
-- Smartcards
-- Windows Hello for Business, if the domain is managed by Microsoft Endpoint Configuration Manager.
+         > [!TIP]
+         > When you connect to the remote PC, enter your account name in this format: AzureAD\yourloginid@domain.com.
 
-In organizations using integrated Active Directory and Azure AD, you can connect from an Azure AD-joined PC to an AD-joined PC when the Azure AD-joined PC is on the corporate network by using any of the following:
+         > [!Note]
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:72]:
```suggestion
         > [!NOTE]
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 7]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
+         > If you cannot connect using Remote Desktop Connection 6.0, you must turn off the new features of RDP 6.0 and revert back to RDP 5.0 by making a few changes in the RDP file. See the details in the [support article](https://support.microsoft.com/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:73]:
```suggestion
         > If you cannot connect using Remote Desktop Connection 6.0, you must turn off the new features of RDP 6.0 and revert back to RDP 5.0 by making a few changes in the RDP file. See the details in this [support article](https://support.microsoft.com/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 8]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
-- Smartcards
-- Windows Hello for Business, if the organization has a mobile device management (MDM) subscription.
-
-In organizations using integrated Active Directory and Azure AD, you can connect from an Azure AD-joined PC to another Azure AD-joined PC by using any of the following:
+## Supported configurations
 
-- Password
-- Smartcards
-- Windows Hello for Business, with or without an MDM subscription.
+The table below lists out the supported configurations for remotely connecting to an Azure AD-joined PC:
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:77]:
```suggestion
The table below lists the supported configurations for remotely connecting to an Azure AD-joined PC:
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 9]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
 
-In organizations using only Azure AD, you can connect from an Azure AD-joined PC to another Azure AD-joined PC by using any of the following:
+| Criteria | RDP from Azure AD registered device| RDP from Azure AD joined device| RDP from Hybrid Azure AD joined device |
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:79]:
```suggestion
| Criteria | RDP from Azure AD registered device| RDP from Azure AD-joined device| RDP from hybrid Azure AD-joined device |
```

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 10]
```diff
@@ -41,57 +42,45 @@ Ensure [Remote Credential Guard](/windows/access-protection/remote-credential-gu
...
+| - | - | - | - |
+| **Client operating systems**| Windows 10 2004 and above| Windows 10 1607 and above | Windows 10 1607 and above |
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:81]:
```suggestion
| **Client operating systems**| Windows 10, version 2004 and above| Windows 10, version 1607 and above | Windows 10, version 1607 and above |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/client-management/connect-to-remote-aadj-pc.md (hunk 7 above)
Comment (illfated) [reply windows/client-management/connect-to-remote-aadj-pc.md:73]: If this had not been a Microsoft Docs document, something more like a personal document, I might have suggested to include the word "this" in the link description. But it is what it is. 🙂

**Comment by illfated** [comment]
I concur. Good job.

File: windows/client-management/connect-to-remote-aadj-pc.md (hunk 2 above)
Comment (RavennMSFT) [reply windows/client-management/connect-to-remote-aadj-pc.md:32]: There are no "-" for Azure AD joined and hybrid Azure AD joined. They are not hyphenated terms

File: windows/client-management/connect-to-remote-aadj-pc.md (hunk 9 above)
Comment (RavennMSFT) [reply windows/client-management/connect-to-remote-aadj-pc.md:79]: Not hyphenated terms

**Comment by RavennMSFT** [comment]
#sign-off

File: windows/client-management/connect-to-remote-aadj-pc.md (hunk 2 above)
Comment (illfated) [reply windows/client-management/connect-to-remote-aadj-pc.md:32]: OK, so maybe a moderate suggestion, containing only the comma suggestions, will be OK instead:
```suggestion
- Your local PC (where you are connecting from) must be either Azure AD joined or Hybrid Azure AD joined if using Windows 10, version 1607 and above, or [Azure AD registered](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-register) if using Windows 10, version 2004 and above. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported. 
```

File: windows/client-management/connect-to-remote-aadj-pc.md (hunk 9 above)
Comment (illfated) [reply windows/client-management/connect-to-remote-aadj-pc.md:79]: Suggesting only the casing correction instead:
```suggestion
| Criteria | RDP from Azure AD registered device| RDP from Azure AD joined device| RDP from hybrid Azure AD joined device |
```

**Comment by RavennMSFT** [comment]
@JohanFreelancer9 , @illfated done with the suggestions

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**